website/docs/introduction/why_riverpod.mdx

```diff
@@ -11,46 +11,53 @@ import { AutoSnippet } from "../../src/components/CodeSnippet";
 Riverpod (anagram of [Provider](https://pub.dev/packages/provider)) is a reactive
 caching framework for Flutter/Dart.
 
-Using declararive and reactive programming, Riverpod is able to take care of
-a large part of your application's logic for you. It can perform network-requests
-with built-in error handling and caching, while automatically re-fetching
-data when necessary.
+Using declarative and reactive programming, Riverpod takes care of a large part of your application's logic by significantly reducing the need for repetitive, boilerplate code. 
```

Comment:
I don't want to talk about boilerplate.

Reply:
That's understood.

Reply:
Will take that feedback into consideration for the next set of PR. Thank you